contrib/jss/invoke.lisp

```diff
@@ -512,28 +545,33 @@ associated is used to look up the static FIELD."
   (when *do-auto-imports* 
     (do-auto-imports)))
 
-(defun japropos (string)
+(defun japropos (string &optional (fn (lambda(match type bundle?) (format t "~a: ~a~a~%" match type bundle?))))
```

Comment:
Kinda convoluted. 

1.  Rewrite as a keyword argument, `&optional` args are hard to refactor when we need further changes.

2. Please either document better.